3-fetch-api-ajax/fetch.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const article = document.querySelector('.main-article');
+const author_img= article.appendChild(document.createElement('img'));
```

Comment:
you don't need to create this element, it can be present on the markup and you just need to use `document.querySelector('.profile-img')` to access it.